_layouts/privacy_policy.html

```diff
@@ -0,0 +1,76 @@
+---
+layout: default
+---
+
+{% assign title = page.title %}
+{% assign content = page.content %}
+
+{% include contactPolicyBanner.html colour="cream" sectionTitle=title description=content classList="flexWrapper padd" %}
+
+<div class="t-one autoHeight success-stories padd simple privacy" uk-scrollspy="target: > div; cls: uk-animation-fade; delay: 100">
+<div class="wrapper">
+    <div class="story-list">
+{% for policy in page.Policies %}
+{% assign remainder = forloop.index | modulo: 2 %}
+<span>
+        <a href="#{{ policy.Language | slugify: "latin" }}" data-nav-lang="{{ policy.Language | slugify: "latin" }}">{{ policy.Language }}</a>
+{% unless forloop.last %}/{% if remainder == 0 %}<br class="brnodisplay">{% endif %}{% endunless %}
+</span>
+{% endfor %}
+    </div>
+
+    <div class="success-story-wrapper">
+{% for policy in page.Policies %}
+<div data-lang="{{ policy.Language | slugify: "latin" }}">
+        {{ policy.Text | markdownify }}
+</div>
+{% endfor %}
+    </div>
+</div>
```

Comment:
all around indentation here is off. I think it makes thinks a lot clearer if any open html element adds indentation and closing removes one.